AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -93,6 +93,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+evantage.gilmoreglobal.com##div[class^="Dialog__backdrop-"]
```

Comment:
cen be hidden by `div[id^="dialog-LTC-"]`

Reply:
Ok. Same thing, but I'll add yours.